examples/serve/https/service.yaml

```diff
@@ -0,0 +1,22 @@
+# SkyServe YAML to run an HTTPS server.
+#
+# Usage:
+#   sky serve up -n https examples/serve/https/service.yaml
+# The endpoint will be printed in the console. You
+# could also check the endpoint by running:
+#   sky serve status --endpoint https
+
+service:
+  readiness_probe: /
+  replicas: 1
+  ssl:
```

Comment:
Call it `tls` instead? Keep align with k8s ingress: https://kubernetes.io/docs/concepts/services-networking/ingress/

Reply:
https://docs.python.org/3/library/ssl.html

Seems like the lib uvicorn used is called `ssl` but supported both SSL and TLS. It is called SSL for historical reasons. Changing to tls now